Index: libstoragemgmt/tools/lsmcli/lsmcli_func.cpp
===================================================================
--- libstoragemgmt.orig/tools/lsmcli/lsmcli_func.cpp	2014-02-24 22:26:47.408088587 +0530
+++ libstoragemgmt/tools/lsmcli/lsmcli_func.cpp	2014-02-24 22:46:58.560064664 +0530
@@ -171,7 +171,7 @@
     int rc = 0;
     lsmInitiator **init = NULL;
     uint32_t num_init = 0;
-    const char format[] = "%-40s%-16s%-5d\n";
+    const char format[] = "%-40s%-16s%-5s\n";
 
     rc = lsmInitiatorList(c, &init, &num_init, 0);
 
